### PR TITLE
mpulse currency converter

### PR DESCRIPTION
**Pull request by fklasen**

### CODE REVIEW

File: edgekv/examples/mpulse-currency-converter/main.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  currencyValue = await edgeKv.getJson({
+    item: currencyCode,
+    default_value: default_value,
+  });
+
+  return currencyValue;
+}
+
+export async function responseProvider(request) {
+  this.request = request;
```
Comment (mryakan) [inline edgekv/examples/mpulse-currency-converter/main.js:31]: Not sure what this is achieving. Why do you believe it is needed?

File: edgekv/examples/mpulse-currency-converter/main.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  try {
```
Comment (mryakan) [inline edgekv/examples/mpulse-currency-converter/main.js:32]: missing default value for finalCurrency. I believe if getCurrency throws an exception (due to getJson throwing one), then finalCurrency will be undefined, which in turn will cause "finalCurrency.value " derefrence to throw an exception.

**Review by mryakan — CHANGES_REQUESTED** [review]

File: edgekv/examples/mpulse-currency-converter/main.js (hunk 1 above)
Comment (fklasen) [reply edgekv/examples/mpulse-currency-converter/main.js]: was probably remains from troubleshooting/collaboration with others, removed it

File: edgekv/examples/mpulse-currency-converter/main.js (hunk 2 above)
Comment (fklasen) [reply edgekv/examples/mpulse-currency-converter/main.js:31]: added

**Review by mryakan — APPROVED** [review]

**Pull request merged**